cmd/deploy.go

```diff
@@ -290,11 +294,19 @@ func installHelmRelease(ctx context.Context, helmClient *helm.Client, releaseNam
 	spinner := ui.NewSpinner("Installing groundcover helm release")
 	spinner.Start()
 	spinner.StopMessage("groundcover helm release is installed")
+	spinner.StopFailMessage("groundcover helm release installation failed")
 	defer spinner.Stop()
 
 	var release *helm.Release
-	if release, err = helmClient.Upgrade(ctx, releaseName, chart, chartValues); err != nil {
-		spinner.StopFailMessage("groundcover helm release installation failed")
+	helmUpgradeFunc := func() error {
```

Comment:
define the funcs outside

Reply:
i need the func on same scope to set release variable, spinner.Poll except func() error